Dockerfile

```diff
@@ -1,13 +1,9 @@
-FROM balena/open-balena-base:v18.0.8
-
-EXPOSE 80
-EXPOSE 81
+FROM balena/open-balena-base:v18.0.8-no-systemd
 
 # hadolint ignore=DL3008
 RUN apt-get update \
 	&& apt-get install -y --no-install-recommends \
 		musl \
```

Comment:
Do we still need musl? I was able to build the image locally with this entire `RUN` step removed so wanted to double-check.